news/2020-09-01-freshers-fair.md

```diff
@@ -0,0 +1,60 @@
+---
+title: The 2020 Fresher's Fair
+---
+
+YUSU have released [a statement](https://www.yorkvision.co.uk/news/exclusive/01/09/2020) about how Freshers' Fair will go forth this year. We have opinions.
```

Comment:
This line hasn't been hard-wrapped to 70 characters, but the link makes that hard.

Reply:
[reference-style link noises] :^)

Reply:
i.e.

```markdown
released [a statement][vision] about

[vision]: https://www.yorkvision.co.uk/news/exclusive/01/09/2020
```